src/main/scala/it/pagopa/interop/tenantprocess/api/impl/TenantApiServiceImpl.scala

```diff
@@ -669,6 +669,7 @@ final case class TenantApiServiceImpl(
     agreements <- agreementManagementService.getAgreements(producerId, consumerId, agreementStates)
     eServices  <- Future.traverse(agreements.map(_.eserviceId))(catalogManagementService.getEServiceById)
     attributeIds = eServices
+      .flatMap(_.descriptors.sortBy(_.version).lastOption.toList)
```

Comment:
This is used to check if a user has the right to verify or revoke an attribute. It was checking if the attribute was present in the eservice, now checks if it's present in the last descriptor (according to the version, as far as they're in the shape "<number>")

Reply:
another chance, but breaking, is to ask for the descriptorId in the API, wdyt?

Reply:
https://github.com/pagopa/interop-be-tenant-process/pull/63#discussion_r1230806981